packages/styles/src/utilities/_variables.scss

```
@@ -31,6 +31,144 @@ $position-values: (
 );
 
 $utilities: (
+  'border': (
+    responsive: true,
+    property: border,
+    class: border,
+    values: (
+      null: var(--post-border-width) var(--post-border-style)
+        rgba(var(--post-border-color-base), var(--post-border-opacity)),
+      0: 0,
+    ),
+  ),
+  'border-top': (
+    property: border-top,
```

Please always use the `block` and `inline` properties instead of the `top`, `bottom`, `right`, and `left` properties:
```suggestion
    property: border-block-start,
```